app/grandchallenge/cases/models.py

```diff
@@ -292,6 +309,24 @@ class Image(UUIDModel):
     color_space = models.CharField(
         max_length=5, blank=False, choices=COLOR_SPACES
     )
+    patient_id = models.CharField(max_length=64, null=True, blank=True)
```

Comment:
You should avoid `null` on char fields (https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.Field.null), remove that and set a default to an empty string for all of the CharFields introduced here.